KWZP2019/KWZP2019/MaintenanceCalendarForm.cs

```diff
@@ -29,5 +33,128 @@ private void btnReturnMain_Click(object sender, EventArgs e)
             this.maintenanceManagementForm.Show();
             this.Hide();
         }
+
+        void populateDataGridView()
+        {
+            this.dgvMaintenance.AutoGenerateColumns = false;  
+            this.dgvMaintenance.DataSource = this.db.Maintenances.ToList<Maintenance>(); 
+        }
+
+        void clear()
+        {
+            this.maintenance.IdMaintenance = 0;
+        }
+
+        private void MaintenanceCalendarForm_Load(object sender, EventArgs e)
+        {
+            clear();
+            populateDataGridView();          
+            List <MaintType> listType = db.MaintTypes.ToList<MaintType>();
+            MaintType recordType = new MaintType();
+            recordType.IdMaintenanceType = -1;
+            recordType.MaintenanceType = "Wszystkie";
+            listType.Add(recordType);
+            comType.DataSource = listType;
+            comType.ValueMember = "IdMaintenanceType";
+            comType.DisplayMember = "MaintenanceType";
+            comType.SelectedValue = -1;
+            List <Machine> listMachines = db.Machines.ToList<Machine>();
+            Machine recordMachine = new Machine();
+            recordMachine.IdMachine = -1;
+            recordMachine.MachineName = "Wszystkie";
+            listMachines.Add(recordMachine);
+            comMachine.DataSource = listMachines;
+            comMachine.ValueMember = "IdMachine";
+            comMachine.DisplayMember = "MachineName";
+            comMachine.SelectedValue = -1;   
+            this.lIsLoading = false;
+            this.machineFilter();
+        }
+
+        private void dgvMaintenance_CellClick(object sender, DataGridViewCellEventArgs e)
+        {
+            if (this.dgvMaintenance.CurrentRow.Index != -1)
+            {
+                try
+                {
+                    this.nIdSelectedMaintenance = Convert.ToInt32(this.dgvMaintenance.CurrentRow.Cells["IdMaintenance"].Value);
+                }
+                catch (Exception ex)
+                {
+                    MessageBox.Show(ex.ToString());
+                }
+            }
+        }
+
+        private void btnDescription_Click(object sender, EventArgs e)
+        {
+            if (this.nIdSelectedMaintenance == 0)
+                return;
+            try
+            {               
+                maintenance = db.Maintenances.Where(maintenance => maintenance.IdMaintenance == this.nIdSelectedMaintenance).First();
+                MaintenanceDescriptionForm maintDescriptionForm = new MaintenanceDescriptionForm(this.db, maintenance.IdMaintDesc, 0);
```

Comment:
ok.. `maint` to skrót od `maintenance` a nie literówka w słowie `main` - właśnie dlatego mówiłem żeby nie używać skrótów... proszę o zmianę na pełne słowo w zmiennych